proposals/0298-asyncsequence.md

```diff
@@ -100,7 +100,7 @@ The standard library will define the following protocols:
 public protocol AsyncSequence {
   associatedtype AsyncIterator: AsyncIteratorProtocol where AsyncIterator.Element == Element
   associatedtype Element
-  func makeAsyncIterator() -> AsyncIterator
+  __consuming func makeAsyncIterator() -> AsyncIterator
 }
 
 public protocol AsyncIteratorProtocol {
```

Comment:
@parkera `AsyncIteratorProtocol` still contains `__consuming mutating func cancel()` on line 109.

Reply:
Yah, thanks. I'm going to fix that with a few other things.